AI/VCAI/AIUtility.cpp

```diff
@@ -126,7 +128,7 @@ const CGHeroInstance * HeroPtr::operator*() const
 	return get();
 }
 
-void foreach_tile_pos(std::function<void(const int3& pos)> foo)
+void foreach_tile_pos(std::function<void(const int3 &pos)> foo)

```

Comment:
maybe the digit "3" is source of bug. However with bugs like this... that source file will get bugged here everytime its processed by uncrustify